apollo/typedefs.ts

```diff
@@ -307,7 +307,7 @@ const typeDefs = gql`
     getAllTalentIds: [ID]!
     getAllUserIds: [ID]!
     getAllEmployerIds: [ID]!
-    getTalentById(id: String!): Talent!
+    getTalentById(id: String!): Talent
```

Comment:
could you change this back to return Talent!
?

Reply:
done